app/src/main/res/values/strings.xml

```diff
@@ -15,7 +15,7 @@
     You should have received a copy of the GNU General Public License
     along with this program. If not, see <http://www.gnu.org/licenses/>.
 -->
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools" tools:locale="en" tools:ignore="MissingTranslation">
```

Comment:
Hmm, I'd rather not ignore missing translations, just for the sake of completing A11y more easily. I could also just add the missing translations.

What do you think, @udenr ?

Reply:
So you would rather copy english original strings into each translation specific xml file when new strings are added?
Otherwise lint is not happy and each check is failing.

Reply:
You could set the MissingTranslation to be treated as warning instead of error. Example [here](https://github.com/SecUSo/privacy-friendly-todo-list/blob/a228ecda2627b85eab79929156274b406bfb1807/app/build.gradle#L56-L58).